controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -269,11 +270,40 @@ func (r *StorageClusterReconciler) reconcileUninstallAnnotations(sc *ocsv1.Stora
 	return nil
 }
 
+// verifyNoStorageConsumerExist verifies there are no storageConsumers exists on the same namespace
+func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.StorageCluster) error {
+
+	if !instance.Spec.AllowRemoteStorageConsumers {
+		return nil
+	}
+
+	storageConsumers := &ocsv1alpha1.StorageConsumerList{}
+	err := r.Client.List(context.TODO(), storageConsumers, &client.ListOptions{Namespace: instance.Namespace})
+	if err != nil {
+		return err
+	}
+
+	if len(storageConsumers.Items) != 0 {
+		err = fmt.Errorf("Can not uninstall storage cluster, StorageConsumers present in the %s Namespace. "+
+			"Clean all StorageConsumers for the uninstallation to proceed", instance.Namespace)
+		r.recorder.ReportIfNotPresent(instance, corev1.EventTypeWarning, "", err.Error())
+		r.Log.Error(err, "Uninstall: Can not uninstall storage cluster because StorageConsumers exists.")
```

Comment:
redundant error messages. Message `Can not uninstall storage cluster because StorageConsumers exists.` is already added in the `err` on line 287